app/repos/people.js

```diff
@@ -374,7 +374,7 @@ const updateContact = async (existingPerson, type, contact, transaction) => {
 
 const deletePerson = async (reference, user, transaction) => {
   if (!transaction) {
-    return sequelize.transaction(async (t) => deletePerson(reference, user, t))
+    return await sequelize.transaction(async (t) => deletePerson(reference, user, t))
   }
```

Comment:
sorry to be a pain, just wondering if you'd be happy to do a quick unit test to check both options on this with a reject.  Don't like putting in awaits on returns unless we have to.  It looks ugly 😃